### PR TITLE
Update prompt.txt to remove the restriction to english

### PR DESCRIPTION
**Pull request by marcelveldt**

### CODE REVIEW

**Comment by OzGav** [comment]
I'm wondering if that line is needed at all? I used a prompt which said all input will be in English and then this
User
3 songs with awesome riffs
ChatGPT
{"media_id": ["Led Zeppelin - Whole Lotta Love", "AC/DC - Thunderstruck", "Guns N' Roses - Sweet Child o' Mine"], "media_type": "track"}
User
drie rocknummers met geweldige riffs
ChatGPT
{"media_id": ["Led Zeppelin - Whole Lotta Love", "AC/DC - Thunderstruck", "Guns N' Roses - Sweet Child o' Mine"], "media_type": "track"}
User
trois chansons rock avec des riffs géniaux
ChatGPT
{"media_id": ["Led Zeppelin - Whole Lotta Love", "AC/DC - Thunderstruck", "Guns N' Roses - Sweet Child o' Mine"], "media_type": "track"}

So three languages, same phrase (according to Google translate!) and same three responses when this was in the prompt "Note that the input query will be in English."

**Comment by marcelveldt** [comment]
sounds valid to me!

File: prompt/prompt.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ Just a track if the artist is not known like this {"media_id":"track name", "med
 
 You must reply with only the JSON model, nothing before nor after because your response will be processed by a search component of a media listening service.
 
-Note that the input query will be in English.
+Note that the input query can be in any language but most likely be English.
```
Comment (marcelveldt) [inline prompt/prompt.txt:28]:
```suggestion
```

File: prompt/prompt.txt [hunk 2]
```diff
@@ -25,7 +25,6 @@ Just a track if the artist is not known like this {"media_id":"track name", "med
 
 You must reply with only the JSON model, nothing before nor after because your response will be processed by a search component of a media listening service.
 
-Note that the input query will be in English.
```
Comment (marcelveldt) [inline prompt/prompt.txt:28]:
```suggestion
```

**Pull request merged**